app/views/moderations/index.html.erb

```diff
@@ -108,11 +109,11 @@
             <% end %>
           </center>
           <div class="body">
-            <%= HTML_Truncator.truncate(
+            <%= raw HTML_Truncator.truncate(
```

Comment:
Hmm any particular reason for this? I'm reading some stuff now and it seems to be a performance improvement, but want to hear your thoughts on the change.

Reply:
First of all, the one subtle difference between raw and html_safe: raw(nil) returns an empty string, while nil.html_safe throws an exception, secondary it's easy for reading.